lib/request.js

```diff
@@ -17,13 +17,15 @@ import {
 import qs from 'qs'
 import routes from './routes'
 import ApiError from './errors'
-import version from './version'
+import { version } from '../package.json'
 
 require('isomorphic-fetch')
 
+const userAgent = (navigator && (`${navigator.userAgent} `)) || ''
+
 const defaultHeaders = {
   'Content-Type': 'application/json',
-  'User-Agent': `pagarme-js/${version}`,
+  'User-Agent': `${userAgent}pagarme-js/${version}`,
```

Comment:
acho que vale a pena adicionar 1 espaço em brancho aqui pra não ficar colado no user agent ja existente
```suggestion
  'User-Agent': `${userAgent} pagarme-js/${version}`,
```

Reply:
tem que ver o que ficou combinado com seals se vamos concatenar apenas no `user-agent` ou no `X-PagarMe-User-Agent`tambem

Reply:
> No segundo commit não tem que rodar um `npm run build` pra ter os arquivos atualizados no `dist`?

Não necessáriamente @otaviopace, o build precisa realmente ser executado apenas quando formos fazer a versão, na minha opinião os arquivos criados pelo build não deveriam ficar no repo, e deveriam ser criados na hora da publicação.

Reply:
> acho que vale a pena adicionar 1 espaço em brancho aqui pra não ficar colado no user agent ja existente

já tem o espaço em branco na const na linha 24, assim evitamos de adicionar um espaço inútil quando por algum motivo obscuro não tivermos um `UserAgent`.